content/en/Getting started/planning.md

```diff
@@ -18,17 +20,127 @@ we can help you schedule pentests with a start date from at least one to three b
+
+### API
+
+We can test both RESTful and GraphQL APIs. However, these APIs work in different ways. While some RESTful APIs can have dozens of endpoints, a GraphQL API has a single endpoint.
+
+If you’re sizing a GraphQL API, identify a list of [queries and mutations](https://graphql.org/learn/queries). For pentest purposes, that’s functionally equivalent to the number of RESTful API endpoints.
+
+To scope a pentest for an API asset, specify the number of the following characteristics of that asset:
+
+- [User Roles](../glossary/#user-role)
+- [Endpoints](../glossary/#api-endpoint) / [GraphQL Mutations](../glossary/#graphql-mutation)
+
+Include those User Roles and Endpoints / GraphQL Mutations that should be in scope for this pentest. If you forget some, you may sacrifice quality in penetration testing.
+
+### External Network
+
+To scope a pentest for an External Network asset, specify the number of public IP addresses to be tested.
+
+### Internal Network
+
+To scope a pentest for an Internal Network asset, specify the number of internal IP addresses to be tested.
+
+If you’re working with servers on the cloud, you can also set up a [Cloud Configuration](#cloud-configuration) asset.
+
+### Cloud Configuration
+
+Cobalt pentesters can test services on the following platforms:
+
+- Google Cloud Platform (GCP)
+- Amazon Web Services (AWS)
+- Microsoft Azure Cloud (Azure)
+
+Each platform includes different categories of services, such as EC2, databases, and machine learning engines.
+
+To scope a pentest for a Cloud Configuration asset, specify the number of the following characteristics of that asset:
+
+- User Accounts / Projects / Subscriptions
+- Unique Services
+
+### Assets of Multiple Types
+
+Sometimes, assets fit into more than one category. To that end, Cobalt supports pentests on assets in the following groups of categories:
+
+- Web + API
+- Web + Internal Network
+- Web + Mobile
+
+To scope a pentest for a combined asset, specify the number of characteristics for each asset type that it includes.
+
+## View Required Credits
+
+Once you’ve identified the [pentest scope](#scope-the-pentest), you can see the number of required [credits](https://www.cobalt.io/pentest-pricing#cobaltcredits) in **Credit(s) Per Pentest**. Whenever you adjust the scope, the number of credits updates. Under **Credits**, you can also view your **Available Balance**.
+
+Note the following:
+
+<ul>
+<li>We accept pentests of at least five (5) credits.</li>
+<li>{{% min-credits-report %}}.</li>
+<li>Pentests requiring more than 20 credits don’t get immediate confirmation. We’ll specify the number of required credits after reviewing the pentest.</li>
+<li>We may adjust the number of credits after reviewing your pentest.</li>
+</ul>
+
 ## Additional Requests
 
-You can also add special requirements for pentesters. For example, if
+You can also specify special requirements for pentesters. For example, if
```

Comment:
You may specific special requirements for pentesters.
Request is not guaranteed and may delay scheduling the pentest.